lib/clamp/command.rb

```diff
@@ -80,6 +80,14 @@ def help
       self.class.help(invocation_path)
     end
 
+    # Get parsed options as Hash
+    #
+    def options
+      @options ||= self.class.declared_options.map do |option|
```

Comment:
Memoising doesn't seem necessary, here, and there's a (albeit slim) chance it could introduce bugs if option values were somehow altered.

Reply:
I can agree, changed.